include/swift/Frontend/FrontendOptions.h

```diff
@@ -243,6 +243,10 @@ class FrontendOptions {
   /// variables by name when we print it out. This eases diffing of SIL files.
   bool EmitSortedSIL = false;
 
+  /// Indices whether the dependency tracker should track system
```

Comment:
Blargh, gonna get this in a follow-up.  I need this patch sooner rather than later.